packages/terra-form-select/src/_Frame.jsx

```diff
@@ -389,11 +398,11 @@ class Frame extends React.Component {
       <div
         {...customProps}
         role={!disabled ? 'combobox' : undefined}
-        aria-controls={!disabled && this.state.isOpen ? 'terra-select-dropdown' : undefined}
+        aria-controls={!disabled && this.state.isOpen ? 'terra-select-menu' : undefined}
```

Comment:
With testing I've been doing with VoiceOver on iOS, to enable usage with the screen reader on mobile, I needed to shift the role and aria attributes from this div to the div with role="textbox" on it on line 406. 

This change matches with [other](https://github.com/cerner/terra-core/issues/1887#issuecomment-460730393) [custom](https://github.com/cerner/terra-core/issues/1887#issuecomment-460734805) [selects](https://github.com/cerner/terra-core/issues/1887#issuecomment-460739869) we've looked. The aria attributes in those code examples are placed on the input element. For this code, the input is essentially the `div[role="textbox"]` element.

Can you test this change and see how it works with JAWS and VoiceOver on desktop.

Reply:
Is that to deal with the focus management? It seems to be reading the results fine on the list on my device with the current role settings.

Reply:
I've switched the code I had back to match similar to DOM structure in this PR. I had run into some issues with aXe with the DOM structure I had tried out.